src/client_side_request.h

```diff
@@ -170,6 +199,13 @@ class ClientHttpRequest
     /// called by StoreEntry when it has more buffer space available
     void resumeBodyStorage();
 
+    /// assigns log_uri with aUri without copying the entire C-string
+    void absorbLogUri(char *aUri);
+    /// resets the current request and log_uri to nil
+    void clearRequest();
+    /// initializes the current unassigned request to the virgin request
```

Comment:
After the method is fixed (see another change request). I would describe it as `sets the current request, asserting that it was unset`. We already use "current request" terminology for describing virgin-or-adapted/redirected request elsewhere, and that terminology works well here.

Please note that, technically, private method descriptions should go into those methods definitions, not declarations. I am not a big fan of that styling rule, so I do not enforce it, but others may. Do not fight them if they do.